src/nrf5_sdk15_platform/environmental/ruuvi_nrf5_sdk15_environmental_mcu.c

```diff
@@ -191,63 +197,71 @@ rd_status_t ri_environmental_mcu_samplerate_get (
 rd_status_t ri_environmental_mcu_resolution_set (
     uint8_t * resolution)
 {
+    const int resolution_set = 10;
```

Comment:
This doesn't really solve the original problem of having a magic constant, it would be preferable to #define the 10. for example `#DEFINE NRF52_TEMP_SENSOR_RESOLUTION 10`

Reply:
Refactored `const int` to the global definition.